adal/src/main/java/com/microsoft/aad/adal/JWSBuilder.java

```diff
@@ -100,6 +106,44 @@ private JwsHeader() {
         }
     }
 
+    /**
+     * Generate generic String key/value pair JWT.
+     *
+     * @param header
+     * @param body
+     * @return String Base64URLSafe(header)+Base64URLSafe(body)
+     * @throws JSONException
+     * @throws UnsupportedEncodingException
+     */
+    public String generateJWT(HashMap<String, String> header, HashMap<String, String> body,
```

Comment:
Input parameter should be `Map` unless there's some specific reason to use `HashMap`

Reply:
ditto

Reply:
done